.circleci/config.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+jobs:
+  build:
+      docker:
+        - image: circleci/node:8.10
+      working_directory: ~/SmartPath-POC/demo-service
+      filters:
+        branches:
+          only: master
+      steps:
+        - checkout
+
+        - run:
+            name: Install Serverless CLI and dependencies
+            command: |
+              sudo npm i -g serverless
+              npm install
+
+        - run:
+            name: Deploy application
+            command: sls deploy
```

Comment:
how are you specifying the aws credentials that you are using to deploy?

Reply:
I added my credentials to AWS permissions in my CircleCI project settings for this project